files/build_templates/docker_image_ctl.j2

```diff
@@ -248,28 +249,18 @@ function postStartAction()
                     $SONIC_CFGGEN -j /etc/sonic/config_db$DEV.json --write-to-db
                 fi
             fi
-
-            if [[ "$BOOT_TYPE" == "fast" ]]; then
```

Comment:
This code section is being removed as it will never get called - the check for `fast` reboot will enter the prev parent if block and not this.

Moreoever, w/ new changes in fast-boot, we have a dedicated flag for fastboot that gets cleared as part of finalizer. The timed expiry mechanism is not needed.

Reply:
Just saw this, this is actually crucial for upgrading from older versions prior fast-reboot enhancements.
For example, upgrading from 202012 images, in case it exceeded 180 seconds no indication of fast-reboot is found in STATE-DB and db-migrator will fail to set fast-reboot new notation.

Reply:
You are right. Thanks for reporting this. The PR was merged when you added comment, and I did not check it.
Fixed this here: https://github.com/sonic-net/sonic-buildimage/pull/16733